.node-version

```diff
@@ -0,0 +1 @@
+5
```

Comment:
Nit: you can also configure your editor to insert a newline before the end of the file. `ensure_newline_at_eof_on_save` is the sublime text setting, I think.